src/app/pages/loading-states/loading-states.component.html

```diff
@@ -0,0 +1,54 @@
+<mat-toolbar color="primary" flexLayout="row">
+    <button *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px;">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <h6>Loading Example</h6>
+    <div fxFlex></div>
+    <button mat-button color="white" (click)="refreshData()">
+        <mat-icon>autorenew</mat-icon>
+    </button>
+</mat-toolbar>
+
+<div style="padding: 20px;">
+    <mat-grid-list [cols]="cols" gutterSize="12px" rowHeight="270px">
+        <mat-grid-tile *ngFor="let device of deviceList">
+            <mat-card class="padding0">
+                <app-placeholder-card *ngIf="!device.data" id="placeholder"></app-placeholder-card>
+
+                <mat-list class="padding0" *ngIf="device.data" id="data-list">
+                    <pxb-hero-banner [divider]="true">
+                        <pxb-hero label="Grade" value="{{ device.data.heroValue }}" units="/100" size="36">
```

Comment:
Progress icons are not being scaled correctly in the Hero (we say 36, but the icons are still 24)...probably an issue with the hero and quite possibly the reason we didn't use progress icons in the example before.